### ISSUE TITLE
Modal's size prop is never used

### ISSUE THREAD

**Issue by Pavek** [reactstrap/reactstrap#57]
So, it has no effect.
 The fix is easy of course. Would you like me to contribute?

**Comment by eddywashere** [comment]
@pavek yes please. Thanks for tracking this. If you can't get to it please let me know and I can add it.

**Comment by TheSharpieOne** [comment]
Fixed in #95
Released in [2.2.0](https://github.com/reactstrap/reactstrap/releases/tag/2.2.0)
Tests added in #100
